frontend/src/container/NewDashboard/DashboardSettings/General/index.tsx

```diff
@@ -57,48 +69,138 @@ function GeneralDashboardSettings(): JSX.Element {
+								onChange={(value: string): void => setUpdatedImage(value)}
+							>
+								{Base64Icons.map((icon) => (
+									<Option value={icon} key={icon}>
+										<img src={icon} alt="dashboard-icon" className="list-item-image" />
+									</Option>
+								))}
+							</Select>
+							<Input
+								data-testid="dashboard-name"
+								className="dashboard-name-input"
+								value={updatedTitle}
+								onChange={(e): void => setUpdatedTitle(e.target.value)}
+							/>
+						</section>
+					</div>
 
-				<div>
-					<Typography style={{ marginBottom: '0.5rem' }}>Description</Typography>
-					<Input.TextArea
-						data-testid="dashboard-desc"
-						rows={5}
-						value={updatedDescription}
-						onChange={(e): void => setUpdatedDescription(e.target.value)}
-					/>
-				</div>
-				<div>
-					<Typography style={{ marginBottom: '0.5rem' }}>Tags</Typography>
-					<AddTags tags={updatedTags} setTags={setUpdatedTags} />
-				</div>
-				<div>
-					<Button
-						style={{
-							margin: '16px 0',
-						}}
-						disabled={updateDashboardMutation.isLoading}
-						loading={updateDashboardMutation.isLoading}
-						icon={<SaveOutlined />}
-						data-testid="save-dashboard-config"
-						onClick={onSaveHandler}
-						type="primary"
-					>
-						{t('save')}
-					</Button>
+					<div>
+						<Typography style={{ marginBottom: '0.5rem' }} className="dashboard-name">
+							Description
+						</Typography>
+						<Input.TextArea
+							data-testid="dashboard-desc"
+							rows={6}
+							value={updatedDescription}
+							className="description-text-area"
+							onChange={(e): void => setUpdatedDescription(e.target.value)}
+						/>
+					</div>
+					<div>
+						<Typography style={{ marginBottom: '0.5rem' }} className="dashboard-name">
+							Tags
+						</Typography>
+						<AddTags tags={updatedTags} setTags={setUpdatedTags} />
+					</div>
+				</Space>
+			</Col>
+			{numberOfUnsavedChanges > 0 && (
+				<div className="overview-settings-footer">
+					<div className="unsaved">
+						<div className="unsaved-dot" />
+						<Typography.Text className="unsaved-changes">
+							{numberOfUnsavedChanges} Unsaved change
+						</Typography.Text>
+					</div>
+					<div className="footer-action-btns">
+						<Button
+							style={{
+								margin: '16px 0',
+							}}
+							disabled={updateDashboardMutation.isLoading}
+							icon={<X size={14} />}
+							onClick={discardHandler}
+							type="text"
+							className="discard-btn"
+						>
+							Discard
+						</Button>
+						<Button
+							style={{
+								margin: '16px 0',
+							}}
+							disabled={updateDashboardMutation.isLoading}
+							loading={updateDashboardMutation.isLoading}
+							icon={<Check size={14} />}
+							data-testid="save-dashboard-config"
+							onClick={onSaveHandler}
+							type="primary"
+							className="save-btn"
+						>
+							{t('save')}
+						</Button>
+					</div>
 				</div>
-			</Space>
-		</Col>
+			)}
+		</div>
```

Comment:
prefer using styles in classname rather than inline-styles

Reply:
follow up PR 😄